src/api/app/services/gitea_status_reporter.rb

```diff
@@ -32,12 +32,12 @@ def call
 
   # TODO: extract to a parent class
   def status_options
-    if @initial_report
-      { context: 'OBS SCM/CI Workflow Integration started',
-        target_url: Rails.application.routes.url_helpers.token_workflow_run_url(@workflow_run.token_id, @workflow_run.id, host: Configuration.obs_url) }
-    elsif @event_type == 'Event::RequestStatechange'
+    if @event_payload[:number]
```

Comment:
Instead of changing this again you could just set the `event_type` in the `SCMStatusReporter` call in the step.

Reply:
I reverted the change here, and added the `event_type: 'Event::RequestStatechange'` to the call, like the following:
`SCMStatusReporter.new(payload, payload, @token.scm_token, workflow_run, event_type: 'Event::RequestStatechange').call`